### PR TITLE
Add Trial Form to Comparison Pages

### PR DESCRIPTION
**Pull request by evantahler**
Screen Shot 2022-01-06 at 12 46 19 PM

This PR also removes manual usage of `<Head>` components on various pages in favor of our `<SEO>` component. 

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: components/TrialForm.tsx [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    </>
+  );
+  const disabled = loading || registered;
+
+  return (
+    <>
+      <Error message={error.generic} />
+      {!registered ? (
+        <Container className="bg-white border shadow border-3 rounded col-11 p-4 mt-4 mt-md-0 me-4 me-lg-0">
+          <h3 className="mx-auto text-center">
```
Comment (andyjih) [inline components/TrialForm.tsx:154]: All of the field labels being centered looks odd to me.
Screen Shot 2022-01-06 at 2 31 54 PM

Can "First Name", "Last Name", "Work Email", and "Company Website" be left-aligned?

**Review by andyjih — CHANGES_REQUESTED** [review]
Looks good overall! One comment to address.

File: components/TrialForm.tsx (hunk 1 above)
Comment (evantahler) [reply components/TrialForm.tsx:154]: Screen Shot 2022-01-06 at 3 01 13 PM
I'm not feeling left-align personally..., especially with other parts of the box being center aligned

Comment (andyjih) [reply components/TrialForm.tsx:154]: What if we left-align everything?

Comment (evantahler) [reply components/TrialForm.tsx:154]: I think that woudn't look great either - the box to me looks like it's 'hovering' above with greater Z-depth, so it sort of wants to be it's own different centered thing.  That's the vibe how it's used on the /trial page too

Comment (andyjih) [reply components/TrialForm.tsx:154]: I feel strongly that the labels should be left-aligned and everything else is centered. It matches what we already do on `/trial`.

Comment (evantahler) [reply components/TrialForm.tsx:154]: Then you've got it! left-aligned via https://github.com/grouparoo/www.grouparoo.com/pull/866/commits/870e4474babddbb8e390326de8354c31fe9fd7eb

**Review by andyjih — APPROVED** [review]

**Pull request merged**